UPGRADE.md

```diff
@@ -1,11 +1,10 @@
 # Yii Widget Upgrading Instructions 
 
-### !!!IMPORTANT!!!
+> This file contains the upgrade notes. These notes highlight changes that could break your
+> application when you upgrade the package from one version to another.
 
-The following upgrading instructions are cumulative. That is,
-if you want to upgrade from version A to version C and there is
-version B between A and C, you need to following the instructions
-for both A and B.
+<div class="alert alert-warning" style="background-color: #e2a03f14;display: flex;padding: 1rem 1rem;margin-bottom: 1rem;border: none;border-left: 3px solid #e2a03f;border-radius: 4px;"><span style="color:red; font-weight: bold;"> **!!! IMPORTANT !!!**<br>The following upgrading instructions are CUMULATIVES. That is, if you want to upgrade from version A to version C and there is  version B between A and C, you need to following the instructions for both A and B.</span>
```

Comment:
Without HTML is better.

Reply:
any github restrictions, difficulties in viewing in some countries or systems, any open source recommendations or just to keep it as simple as possible?
Please explain why you think so

Reply:
It's difficult when see it in text mode.

Reply:
done